src/web/app/pages-instructor/instructor-search-page/instructor-search-page.component.ts

```diff
@@ -18,6 +16,24 @@ export interface SearchStudentsTable {
   sections: StudentListSectionData[];
 }
 
+/**
+ * Data object for communication with child comment result component
+ */
+export interface SearchCommentsTable {
+  courseId: string;
+  sessionName: string;
+  sections: CommentListSectionData[];
+}
+
+/**
+ * Parameters inputted by user to be used in search
+ */
+export interface SearchParams {
```

Comment:
The type definition should be in `src/web/app/pages-instructor/instructor-search-page/instructor-search-bar/instructor-search-bar.component.ts` right?